src/server/debug.rs

```diff
@@ -386,6 +388,101 @@ impl Debugger {
         Ok(())
     }
 
+    pub fn remove_region(&self, region_id: u64) -> Result<()> {
+        let kv = self.engines.kv_engine.as_ref();
+        let raft = self.engines.raft_engine.as_ref();
+        let kv_wb = WriteBatch::new();
+        let raft_wb = WriteBatch::new();
+
+        let key = keys::raft_state_key(region_id);
+        let raft_state = match box_try!(raft.get_msg::<RaftLocalState>(&key)) {
+            Some(s) => s,
+            None => return Err(Error::Other("No RaftLocalState in RAFT rocksdb".into())),
+        };
+        box_try!(clear_meta(
+            kv,
+            raft,
+            &kv_wb,
+            &raft_wb,
+            region_id,
+            &raft_state
+        ));
+
+        let mut write_opts = WriteOptions::new();
+        write_opts.set_sync(true);
+        box_try!(kv.write_opt(kv_wb, &write_opts));
```

Comment:
Tombstone state is not set.

Reply:
Tombstone it is not my purpose. I think we need more comments for it.

Reply:
I have commented `fn remove-region`. Maybe it's better now.